### ISSUE TITLE
AttributeError: 'DiGraph' object has no attribute 'node'

### ISSUE THREAD

**Issue by albertca** [vinci1it2000/formulas#40]
This is using formulas 0.4.0 under python 3.7.3.

Steps to reproduce:

```
pip install formulas
python
>>> import formulas
>>> formulas.get_functions()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/albert/.virtualenvs/formulas/lib/python3.7/site-packages/formulas/functions/__init__.py", line 310, in get_functions
    functions.update(importlib.import_module(name, __name__).FUNCTIONS)
  File "/home/albert/.virtualenvs/formulas/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/albert/.virtualenvs/formulas/lib/python3.7/site-packages/formulas/functions/eng.py", line 142, in <module>
    FUNCTIONS[k] = wrap_func(hex2dec2bin2oct(k))
  File "/home/albert/.virtualenvs/formulas/lib/python3.7/site-packages/formulas/functions/eng.py", line 133, in hex2dec2bin2oct
    _func = sh.DispatchPipe(dsp, function_id, [i, 'places'], [o])
  File "/home/albert/.virtualenvs/formulas/lib/python3.7/site-packages/schedula/utils/dsp.py", line 1081, in __init__
    self.pipe = self._set_pipe()
  File "/home/albert/.virtualenvs/formulas/lib/python3.7/site-packages/schedula/utils/dsp.py", line 1097, in _set_pipe
    return [_make_tks(*v['task'][-1]) for v in self._sol.pipe.values()]
  File "/home/albert/.virtualenvs/formulas/lib/python3.7/site-packages/schedula/utils/sol.py", line 345, in pipe
    return get_full_pipe(self)
  File "/home/albert/.virtualenvs/formulas/lib/python3.7/site-packages/schedula/utils/alg.py", line 462, in get_full_pipe
    sub_sol = s.workflow.node[path[-1]]['solution']
AttributeError: 'DiGraph' object has no attribute 'node'
```

**Comment by albertca** [comment]
Downgrading networkx==2.3 fixes the problem. So the issue starts at the recently released networkx==2.4:

https://pypi.org/project/networkx/#history

**Comment by ankostis** [comment]
See vinci1it2000/schedula#17.

**Comment by vinci1it2000** [comment]
This issue has been fixed with the release of schedula==0.3.6.
